irohad/ametsuchi/impl/block_index.hpp

```diff
@@ -18,12 +18,22 @@
 #ifndef IROHA_BLOCK_INDEX_HPP
 #define IROHA_BLOCK_INDEX_HPP
 
+#include "shared_model/utils/polymorphic_wrapper.hpp"
+
+namespace shared_model {
+  namespace interface {
+    class Block;
+  }
+}
 namespace iroha {
   namespace model {
```

Comment:
I think it should not be here

Reply:
Why? I used forward declaration in order to skip including the whole file

Reply:
But old model is no longer used. Or I missed something?

Reply:
Ah, you are talking about 28+ line, got It
Agree